diff --git a/src/marionette.rs b/src/marionette.rs
index 87657ba..51af7e0 100644
--- a/src/marionette.rs
+++ b/src/marionette.rs
@@ -1264,7 +1264,7 @@ impl MarionetteConnection {
     }
 
     pub fn connect(&mut self, browser: &mut Option<Browser>) -> WebDriverResult<()> {
-        let timeout = time::Duration::from_secs(60);
+        let timeout = time::Duration::from_secs(600);
         let poll_interval = time::Duration::from_millis(100);
         let now = time::Instant::now();
 
